CubeTest/Graphics.cs

```diff
@@ -193,7 +187,7 @@ public static unsafe class Graphics {
 		
 		//Create our device
 		WebGPU.AdapterRequestDevice(Adapter, new DeviceDescriptor {
-			RequiredFeaturesCount = 1,
+			// RequiredFeaturesCount = 1,
 			RequiredFeatures      = &name,
```

Comment:
RequiredFeaturesCount was renamed to RequiredFeatureCount, but in this case its fine, since we arent using the timestamp query code, at some point we should properly re-implement timestamp queries for profiling